### DISCUSSION TITLE
Release v1.4.2 / 5.59.2

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2328] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes various compatibility and usability issues.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog
### Added
- added tooltips to resource access modes [#2300](https://github.com/sandboxie-plus/Sandboxie/issues/2300)
- added UI option to control ApplyElevateCreateProcessFix [#2302](https://github.com/sandboxie-plus/Sandboxie/issues/2302)
- added message 2226 issued when a process may need 'ApplyElevateCreateProcessFix=y'

### Changed
- Moved Restrictions tab from the new Security page back to the General page

### Fixed
- fixed wrong button captions in the breakout process page
- fixed issue with saving box recovery options
- fixed the display problem of Sandman [#2306](https://github.com/sandboxie-plus/Sandboxie/pull/2306) (thanks okrc)
- fixed theme not auto changing [#2307](https://github.com/sandboxie-plus/Sandboxie/pull/2307)
- fixed issue with saving sandman window state on windows shutdown
- fixed miscellaneous minor issues [#2301](https://github.com/sandboxie-plus/Sandboxie/issues/2301)
- fixed issue with MSEdge introduced in 106.x [#2325](https://github.com/sandboxie-plus/Sandboxie/issues/2325)
- fixed vivaldi hooking issue, UseVivaldiWorkaround is no longer needed [#1783](https://github.com/sandboxie-plus/Sandboxie/issues/1783)
- fixed issues with misc advanced options page [#2315](https://github.com/sandboxie-plus/Sandboxie/issues/2315)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.4.2'>Release v1.4.2 / 5.59.2</a>.</em>

**Comment by BilleBarrett** [comment]
starting to get these
chrome.exe (12936): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000002; MsgId: 9 - chrome.exe [C0000034]
What did I check that I shouldn't have :-)
I MUST HAVE SOMETHING SET WRONG ??
Today 10/13/2022
msedge.exe (4648): SBIE2308 Could not create object directory:  [11 / 00000005]
msedge.exe (10704): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000002; MsgId: 9 - msedge.exe [C0000034]

**Reply by PanoWest** [reply]
Similar problem using Firefox with Sandboxie classic 5.59.2. Primarily happens if I go to anything Google or YouTube.
SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]
Not sure if its related.

**Reply by isaak654** [reply]
See if this helps (Firefox only): https://github.com/sandboxie-plus/Sandboxie/issues/2356#issuecomment-1283731833

**Reply by PanoWest** [reply]
> See if this helps 

Yes that seems to have worked. Thanks.

**Comment by ch38shm** [comment]
win11 22H2，can't run explorer.exe in sandboxie(error： SBIE2335 )，and I find that some software in it can't  create file

**Reply by isaak654** [reply]
A system restart may be required after installing Sandboxie on Windows 11.

**Comment by BilleBarrett** [comment]


**Reply by isaak654** [reply]
On Win11 22H2, Explorer.exe only fails for me in the hardened boxes (#2341), while I didn't encounter any issue in the standard sandbox type (yellow box icon).

**Comment by lupolupp** [comment]
I use Edge as default browser. With the new version of Sandboxie my startpages are not longer opened automatically. I have two startpages created in the native, unsandboxed Edge, where the pages will be opened automatically without an issue. But when I start Edge with Sandboxie, only the startpage from Microsoft will be openend.
Sandboxie has full direct access to my favorites, the bookmarks and the whole profile of Edge. 
By closing sandboxed Edge, all sandboxed data will automatically be deleted, by restarting Edge all options should be like the unsandboxed Edge. So what could be the problem?
Thank you very much!
Lupo

**Comment by BilleBarrett** [comment]
Why am I getting these error messages, Please point me in the right directions so I can stop them 
14:54:19.029   chrome.exe (2864): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 9 - chrome.exe [C0000034]

**Comment by BilleBarrett** [comment]
msedge.exe (5636): SBIE2308 Could not create object directory:  [11 / 00000005] is the only browser getting error since I updated to windows 11 2H22

**Reply by isaak654** [reply]
Same as #2342 (no news yet)

**Comment by BilleBarrett** [comment]
I have to transfer files to my clients and lately this happen when ever I start a transfer
chrome.exe (8724): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 9 - chrome.exe [C0000034]
Here is my Sandoxie.ini files 
[Sandboxie-Plus.ini.zip](https://github.com/sandboxie-plus/Sandboxie/files/9810911/Sandboxie-Plus.ini.zip)
